modelservice/src/main/java/com/expedia/adaptivealerting/modelservice/repo/impl/DetectorMappingRepositoryImpl.java

```diff
@@ -225,6 +226,20 @@ public void deleteDetectorMapping(String id) {
         }
     }
 
+    @Override
+    public void deleteMappingByDetectorUUID(UUID uuid) {
+        SearchMappingsRequest searchMappingsRequest = new SearchMappingsRequest();
```

Comment:
We can replace return types of local variables with lombok's val so that the consistency is maintained.